source/framework/vx_exemplar.c

```diff
@@ -252,6 +255,18 @@ static vx_reference ownCreatePyramidFromExemplar(
     pmd = vxCreatePyramid(context, levels, scale, width, height,
             format);
 
+    if ((vx_status)VX_SUCCESS == vxGetStatus((vx_reference)pmd))
+    {
+        vx_uint32 i;
+        vx_status status = (vx_enum)VX_SUCCESS;
```

Comment:
Can you please update this to a (vx_status) cast?

Reply:
I have made this change as part of my latest rebase locally.  It will reflect once I push it back to bosch branch